src/com/library/backlog/Library.java

```diff
@@ -0,0 +1,213 @@
+package com.library.backlog;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import com.library.userroles.UserRole;
+public class Library {
+    
+    String isbn;
+    String title;
+    Book book;
+    ArrayList<User> userList;
+    private UserRole role;
+   // String usernameEntered;
+    String passwordEntered;
+    private List<Book> bookList;
+    // actually we doing this here:
+    Scanner scanner = new Scanner(System.in);
+    // List<Book> bookList = new ArrayList<Book>(); 
+    // List is an interface and ArrayList is the implementation of it, List gives more flexibility than an ArrayList and less bugs
+    
+    
+    
+    public Library(){
+        //@Mou-Comment on calling the class just register the users
+        bookList = new ArrayList<Book>();
+        userList = new ArrayList<User>();
+        addNewUser(UserRole.ADMIN, "Tayyab", "adminpass");
```

Comment:
UI code in the Library app

You can clean POJOs in your other objects, expose public methods like

addBook(Book)
checkout(Book book, User user)
checkin(Book)

these operations should then be accessed via the LibraryApp.class